src/main/java/restaurant_screens/RestaurantController.java

```diff
@@ -18,14 +18,14 @@ public RestaurantController(RestaurantInputBoundary inputData) {
         this.userInput = inputData;
     }
 
-    RestaurantResponseModel create(String owner, String name, String location, String cuisineType, int priceBucket) {
+    RestaurantResponseModel create(OwnerUser owner, String name, String location, String cuisineType, int priceBucket) {
         RestaurantRequestModel requestModel = new RestaurantRequestModel(owner, name, location,
                 cuisineType, priceBucket);
 
         return userInput.create(requestModel);
     }
 
-    RestaurantResponseModel edit(String owner, String name, String location,
+    RestaurantResponseModel edit(OwnerUser owner, String name, String location,
```

Comment:
Why the switch to OwnerUser, how will this affect previous implementations and the restaurant database? You  take in owner there too, no?

Reply:
The restaurant database still stores the owner ID as the latest implementation takes in Restaurant rather than a DSrequestModel, however this is not affected by this, as the Owner is only passed through to allow for manipulation of the ownedRestaurants and to be able to update the user database.